meinberlin/assets/scss/components/_header.scss

```diff
@@ -43,12 +47,34 @@
 
 .main-nav__logo {
     img {
-        width: 7em;
-        margin-top: -3em;
-        margin-bottom: -3em;
+        width: 6em;
+        margin-top: -2em;
+        margin-bottom: -2em;
     }
 }
 
 .main-nav__item {
     padding: 0.2em 0.4em;
 }
+
+.dialogue-box {
+    display: flex;
+    margin-top: -10px;
+
+    > div {
+        height: 43px;
+        background-image: url('/static/images/dialoguebox.png');
+        background-repeat: no-repeat;
+    }
+}
```

Comment:
It would be more robust to use a separate image for left/right that just contains the line and repeats along the x-axis.